### PR TITLE
adding max by and min by function

### PR DESCRIPTION
**Pull request by Lordworms**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -26,14 +26,14 @@ use datafusion::logical_expr::AggregateUDF;
 #[macro_use]
 pub mod macros;
 pub mod common;
+pub mod max_min_by;
 pub mod mode;
-
 pub mod expr_extra_fn {
     pub use super::mode::mode;
```
Comment (s5dsn-eqee) [inline src/lib.rs:34]: Pls add 
```
pub use super::min_max::max;
pub use super::min_max::min;
```

**Review by s5dsn-eqee — COMMENTED** [review]
Thank you @Lordworms 

Could you add information about the function to the README.

File: src/lib.rs (hunk 1 above)
Comment (Lordworms) [reply src/lib.rs:34]: Sure.

**Comment by Lordworms** [comment]
> Thank you @Lordworms
> 
> Could you add information about the function to the README.

Got it

**Comment by s5dsn-eqee** [comment]
lgtm
Thanks @Lordworms

**Pull request merged**